src/app/cube/old-details/details.component.ts

```diff
@@ -163,29 +165,38 @@ export class DetailsComponent implements OnInit, OnDestroy {
 
     try {
       this.resetRatings();
-      this.releasedLearningObject = await this.learningObjectService.getLearningObject(
-        author,
-        name
-      );
 
-      // FIXME: This filter should be removed when service logic is updated
-      this.releasedChildren = this.releasedLearningObject.children.filter(
-        child => {
-          return child.status === LearningObject.Status['RELEASED'] ||
-            child.status === LearningObject.Status['REVIEW'] ||
-            child.status === LearningObject.Status['PROOFING'] ||
-            child.status === LearningObject.Status['WAITING'];
-        }
-      );
+      const resources = ['children', 'parents', 'outcomes', 'materials', 'metrics', 'ratings'];
+      this.uriRetrieverService.getLearningObject({author, name}, resources).pipe(takeUntil(this.isDestroyed$)).subscribe(async (object) => {
+        if (object) {
+          this.releasedLearningObject = object;
+          
+          // FIXME: This filter should be removed when service logic is updated
```

Comment:
What service logic needs to be updated?

Reply:
I don't know, this code was copied from the old service and put into the subscribe of the new service.